datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -6739,6 +6749,36 @@ input AddOwnersInput {
     resourceUrn: String!
 }
 
+"""
+Input provided when adding owners to a batch of assets
+"""
+input BatchAddOwnersInput {
+    """
+    The primary key of the owners
+    """
+    owners: [OwnerInput!]!
+
+    """
+    The target assets to attach the owners to
+    """
+    resources: [ResourceRefInput]!
+}
+
+"""
+Input provided when removing owners from a batch of assets
+"""
+input BatchRemoveOwnersInput {
+    """
+    The primary key of the owners
+    """
+    ownerUrns: [String!]!
+
+    """
+    The target assets to remove the owners from
+    """
+    resources: [ResourceRefInput]!
```

Comment:
should `ResourceRefInput` be required here?

Reply:
I want to keep the 'resources' input semantics across all batch endpoints... This would allow us to extend these concepts to column in the future without any model changes... but that's not a clear advantage i agree